### ISSUE TITLE
UI: Update colors of battery icons

### ISSUE THREAD

**Issue by matthew-ia** [TeamSunergy/chase-ui#5]
The icons should be colored depending on their SOC level. Something like this:
![widget_primary_cluster 2x](https://user-images.githubusercontent.com/17885141/41481831-6f6a9346-70a1-11e8-821e-1b0c1b4f8bbc.png)

**Comment by stevesoaress** [comment]
@matthew-ia Was this feature implemented?

**Comment by matthew-ia** [comment]
@stevesoaress Nope, I went with a slightly different icon. Based on the battery level, an icon is set that indicates the level (like a normal battery status indicator). ~~Those icons can easily be colored, as they're vectors from Google's Material Design icon set (all the icons I implemented are from that library).~~ Those icons wouldn't work like the rest of the icons (loading them from Google's material design icon library; these are actually downloaded as local files), but they can still be colored using a vector editing program such as Adobe Illustrator.
